### PR TITLE
[TESTENG-1457] Updating documentation references to the public URI.

### PR DESCRIPTION
**Pull request by paulletang**
Complete the following tasks before merging in your PR:
- [x] If this is a user facing change, update [version.txt](https://github.com/relativitydev/relativity.testing.framework/blob/master/version.txt) and [CHANGELOG.md](https://github.com/relativitydev/relativity.testing.framework/blob/master/CHANGELOG.md)

### CODE REVIEW

File: source/Relativity.Testing.Framework.Documentation/index.md [hunk 1]
```diff
@@ -3,4 +3,4 @@ Welcome to the Relativity Testing Framework documentation!
 
 Here you can find documentation and guides on common use cases for Relativity Testing Framework.
 
-Also check of the [Relativity Testing Framework Api documentation](https://glowing-spork-1e23a31b.pages.github.io/api/index.html).
\ No newline at end of file
+Also check of the [Relativity Testing Framework Api documentation](https://relativitydev.github.io/relativity.testing.framework.api/).
```
Comment (briantoms) [inline source/Relativity.Testing.Framework.Documentation/index.md:6]: `check of` -> `check out`.

File: source/Relativity.Testing.Framework.Documentation/index.md [hunk 2]
```diff
@@ -3,4 +3,4 @@ Welcome to the Relativity Testing Framework documentation!
 
 Here you can find documentation and guides on common use cases for Relativity Testing Framework.
```
Comment (briantoms) [inline source/Relativity.Testing.Framework.Documentation/index.md:4]: Are we going with `Relativity Testing Framework` or `Relativity.Testing.Framework`?

Comment (paulletang) [reply source/Relativity.Testing.Framework.Documentation/index.md:4]: We should try to standardize on `Relativity.Testing.Framework`.
I switch back and forth more than I should. :/

Comment (paulletang) [reply source/Relativity.Testing.Framework.Documentation/index.md:4]: I don't mind fixing these two issues here, but I was trying to keep the scope of this to just the URI change.
We should come back and standardize the verbiage though.

**Review by briantoms — DISMISSED** [review]

File: source/Relativity.Testing.Framework.Documentation/index.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Relativity Testing Framework
```
Comment (briantoms) [inline source/Relativity.Testing.Framework.Documentation/index.md:1]: Let's at least be consistent within the file 😄

Comment (paulletang) [reply source/Relativity.Testing.Framework.Documentation/index.md:1]: A good starting point. 😄

Comment (briantoms) [reply source/Relativity.Testing.Framework.Documentation/index.md]: I didn't want to make another comment on the line immediately following this one (I feel like no one likes the "here too" comments) but maybe I should have in this case.

Comment (paulletang) [reply source/Relativity.Testing.Framework.Documentation/index.md]: Yeah, I'm not a big fan of those comments.

Without being focused on the scope of the future story, it wasn't on my mind.

**Review by jonatanjakoniuk — APPROVED** [review]

**Pull request merged**